src/registrar/admin.py

```diff
@@ -547,6 +549,23 @@ class DomainApplicationAdmin(ListHeaderAdmin):
 
     """Custom domain applications admin class."""
 
+    class InvestigatorFilter(admin.SimpleListFilter):
+        """Custom investigator filter that only displays users with the manager role"""
+
+        title = "investigator"
+        parameter_name = "investigator"
+
+        def lookups(self, request, model_admin):
+            """Lookup reimplementation, gets users of is_staff.
+            Returns a list of tuples consisting of (user.id, user)
+            """
+            privileged_users = User.objects.filter(is_staff=True)
+            return [(user.id, user) for user in privileged_users]
+
+        def queryset(self, request, queryset):
+            """Custom queryset implementation, filters by investigator"""
+            return queryset.filter(investigator=self.value())
+
```

Comment:
This reduces what we can filter by, but it implies that we don't expect investigators to be users that aren't is_staff. This may be a model level change, but I'm taking the conservative approach and am not changing the model definition as that was not explicit in the ticket.

Would this be better to do it from the model level? I tested it and validation can be done on save() to check for is_staff